### PR TITLE
Update deployment.md

### PR DESCRIPTION
**Pull request by apecollector**
Fix typo and consistently use a single question mark.

The Redwood v1 release candidate has been published, and we're actively working toward a 1.0.0 GA. 🎯

Read @mojombo's post about what to expect, what this means for current priorities, and how to help:
- v1-rc Announcement: https://community.redwoodjs.com/t/what-the-1-0-release-candidate-phase-means-and-when-1-0-will-drop/2604
- v1-rc Priorities that need community help: https://redwoodjs.com/good-first-issue

New Issues and PRs are always welcome and helpful! 😀 We will respond and prioritize. If you need help faster than what we can provide here, you can reach out via the community chat or forums:
- Discord Chat: https://discord.gg/redwoodjs
- Discourse Forum: https://community.redwoodjs.com/

**When you're ready to proceed, just delete this message.**

### CODE REVIEW

**Comment by jtoar** [comment]
@apecollector thanks for the typo fix! I think the doubling up on the question marks was a stylistic flourish to emphasize outrageousness. Ok if we keep that? Or did it detract from the reading?

File: docs/docs/tutorial/chapter4/deployment.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ If the deploy failed, check the log output in Netlify and see if you can make se
 
 #### Custom Subdomain
 
-You can customize the subdomain that your site is published at (who wants to go to `agitated-mongoose-849e99.netlify.app`??) by going to **Site Settings > Domain Management > Domains > Custom Domains**. Open up the **Options** menu and select **Edit site name**. Your site should be available at your custom subdomain (`redwood-tutorial.netlify.app` is much nicer) almost immediately.
+You can customize the subdomain that your site is published at (who wants to go to `agitated-mongoose-849e99.netlify.app`?) by going to **Site Settings > Domain Management > Domains > Custom Domains**. Open up the **Options** menu and select **Edit site name**. Your site should be available at your custom subdomain (`redwood-tutorial.netlify.app` is much nicer) almost immediately.
```
Comment (thedavidprice) [inline docs/docs/tutorial/chapter4/deployment.md:132]:
```suggestion
You can customize the subdomain that your site is published at (who wants to go to `agitated-mongoose-849e99.netlify.app`??) by going to **Site Settings > Domain Management > Domains > Custom Domains**. Open up the **Options** menu and select **Edit site name**. Your site should be available at your custom subdomain (`redwood-tutorial.netlify.app` is much nicer) almost immediately.
```

**Comment by thedavidprice** [comment]
@apecollector @jtoar I added the ?? back 'cause if I know Rob it was intentional.

Can remove if we feel it's truly a distraction. Just wanted to get these edits in while I was working through PRs.

PS Welcome to Redwood!

**Comment by apecollector** [comment]
Yeah that's fine, wasn't sure if it was intentional or not since other questions only had a single question mark and I've never seen that style before.

**Pull request merged**